clients/cpp-console/RaspberryPi.md

```diff
@@ -40,12 +40,13 @@ There are many ways to do development on a Raspberry pi. It may be useful to tak
 
   ```sh
   ln -s Cognitive-Services-Voice-Assistant/samples/clients/cpp-console
```

Comment:
Also might as well fix these paths here. There is no samples folder anymore

Reply:
Doing another commit for this. Thanks!